data/sql/updates/pending_db_world/rev_1646749877159440400.sql

```diff
@@ -0,0 +1,389 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1646749877159440400');
+
+-- ¡Encuentra a Kurun! - ID 11981
+DELETE FROM `quest_offer_reward_locale` WHERE `ID`=11981 AND `locale` IN ('esES', 'esMX');
+INSERT INTO `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) VALUES
```

Comment:
The texts on the other hand, are grouped by missions, along with the objectives. I understand that if we do it the way you suggest, it would be less SQL queries and possibly more efficient. But at the end of the day, when all these files are moved to the database, those queries will become obsolete, won't they? In other words, when they are passed to base, they would be optimized so to speak. Or am I wrong?

Reply:
I'm going to try, anyway. It's going to be fun.